packages/timeline-state-resolver-types/src/__tests__/library.spec.ts

```diff
@@ -1,4 +1,5 @@
 // These imports are pointed to what external libraries will import
+// eslint-disable-next-line
```

Comment:
This hasn't come up in the CI as an issue so I don't think there should be an ignore here.

Reply:
I added this line due to getting a linting error (`no-extraneous-dependencies`) locally in the pre-commit script.
But you're right, it must have been something in my local setup, as I can't reproduce the issue on another system.